modules/support-collecting-network-trace.adoc

```diff
@@ -0,0 +1,96 @@
++
+----
+$ oc get nodes
+----
+
+. Enter into a debug session on the target node. This step instantiates a debug pod called `<node_name>-debug`:
++
+----
+$ oc debug node/my-cluster-node
+----
+
+. Once the debug command prompt appears, set a new root directory within the shell to `/host` so that the node's binaries can be used:
++
+----
+# chroot /host
+----
+
+[NOTE]
+====
+{product-title} {product-version} cluster nodes running Red Hat Enterprise Linux CoreOS are immutable and rely on Operators to apply cluster changes. Accessing cluster nodes using SSH is not recommended and nodes will be tainted as _accessed_. However, if the OpenShift API is not available, or `kubelet` is not properly functioning on the target node, `oc` operations will be impacted. In such situations, it is possible to access nodes using `ssh core@<node>.<cluster_name>.<sub_domain>.<domain>` instead.
+====
+
+. From within the `chroot` environment console, obtain the node's interface names:
++
+----
+# ip ad
+----
+
+. Start a `toolbox` container, which includes the required binaries and plug-ins to run `sosreport`:
++
+----
+# toolbox
+----
++
+[NOTE]
+====
+If an existing `toolbox` pod is already running, the `toolbox` command outputs `'toolbox-' already exists. Trying to start...`. Remove the running toolbox container with `podman rm toolbox-` and spawn a new toolbox container, to avoid `tcpdump` issues.
+====
+
+. Initiate a `tcpdump` session on the cluster node and redirect output to a capture file. This example uses 'ens5' as the interface name:
++
+----
+tcpdump -nn -s 0 -i ens5 -w /host/var/tmp/my-cluster-node_$(date +%d_%m_%Y-%H_%M_%S-%Z).pcap  <1>
+----
+<1> The `tcpdump` archive's file path is outside of the `chroot` environment, because the `toolbox` container mounts the node at `/host`.
+
+. If a `tcpdump` capture is required for a specific container on the node, follow these steps:
+.. Determine the target container ID:
++
+----
+# chroot /host crictl ps
+----
++
+.. Deterine the container's process ID. In this example, the container ID is 'a7fe32346b120':
++
+----
+# chroot /host crictl inspect a7fe32346b120 --output yaml | grep 'pid:' | awk '{print $2}'
+----
++
+.. Initiate a `tcpdump` session on the container and redirect output to a capture file. This example uses 'ens5' as the interface name and '49628' as the container's process ID:
++
+----
+# nsenter -n -t 49628 -- tcpdump -nn -i ens5 -w /host/var/tmp/my-cluster-node-my-container_$(date +%d_%m_%Y-%H_%M_%S-%Z).pcap.pcap  <1>
+----
+<1> The `tcpdump` archive's file path is outside of the `chroot` environment, because the `toolbox` container mounts the node at `/host`.
+
+. Extract the `tcpdump` capture file from the cluster node and provide it to Red Hat Support for analysis, using one of the following methods:
++
+* Run `redhat-support-tool` to attach the file directly to an existing Red Hat support case, from within the toolbox container. This example uses support case ID '01234567':
```

Comment:
s/Red Hat support/Red Hat Support/
Also, I'd start the sentence with, "From within the toolbox container...."

Reply:
Thanks Bob. Resolved.